app/src/main/java/org/mozilla/fenix/search/telemetry/Utils.kt

```diff
@@ -22,6 +22,21 @@ internal fun getTrackKey(
 
     if (provider.codeParam.isNotEmpty()) {
         code = uri.getQueryParameter(provider.codeParam)
```

Comment:
I can imagine us assigning this to a `codeParam` variable. I wonder how far that could get us without introducing the `originalCode`, but continuing to modify `code`.

Reply:
I don't quite get. Do you mean adding the "_" after the `codeParam`, or adding other parameters in `SearchProviderModel` apposed with `codeParam`?